packages/flutter/lib/src/material/text_field.dart

```diff
@@ -351,6 +352,7 @@ class TextField extends StatefulWidget {
     this.autofocus = false,
     this.obscuringCharacter = '•',
     this.obscureText = false,
```

Comment:
I think these other instances of `obscureText` should be @Deprecated as well, right?

Reply:
Also in editable_text.dart, cupertino/text_field.dart, and anywhere else.